### ISSUE TITLE
event feed keyboard shortcut is not accessible

### ISSUE THREAD

**Issue by susanev** [chef/automate#3829]
## overview
the keyboard shortcut to reset the timescale on the event feed is not accessible because users cannot turn it off or remap it. to solve this, and because there is a button that does this already, let's remove the keyboard short cut. in the future, if we want to support keyboard shortcuts across automate it will need to be a full feature that allows folks to turn them off and remap them as needed.

## definition of done
- [ ] remove keyboard shortcut on event feed
- [ ] remove keyboard shortcut info from subheading on the event feed
- [ ] remove keyboard shortcut mentions from docs https://automate.chef.io/docs/event-feed/
- [ ] update all event feed screenshots, also work with docs, we prolly only need one on that page
